aten/src/ATen/Context.cpp

```diff
@@ -529,6 +529,10 @@ void Context::setDisplayVmapFallbackWarnings(bool enabled) {
   display_vmap_fallback_warnings_ = enabled;
 }
 
+bool Context::isDefaultMobileCPUAllocatorSet() {
+  return prev_allocator_ptr_ != nullptr;
+}
+
```

Comment:
I can move the changes related to the MobileCPUAllocator to a different PR if needed.

Reply:
Nah it's good